homeassistant/components/synology_dsm/config_flow.py

```diff
@@ -58,11 +57,11 @@ def _discovery_schema_with_defaults(discovery_info: DiscoveryInfoType) -> vol.Sc
     return vol.Schema(_ordered_shared_schema(discovery_info))
 
 
-def _reauth_schema_with_defaults(user_input: dict[str, Any]) -> vol.Schema:
+def _reauth_schema() -> vol.Schema:
     return vol.Schema(
         {
-            vol.Required(CONF_USERNAME, default=user_input.get(CONF_USERNAME, "")): str,
-            vol.Required(CONF_PASSWORD, default=user_input.get(CONF_PASSWORD, "")): str,
+            vol.Required(CONF_USERNAME, default=""): str,
```

Comment:
Why do we need defaults if they are just empty strings?

Reply:
uhh ... it was already late 🙈